eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -70,7 +70,7 @@ library
 
     build-depends:
         base >=4.14 && <5,
-        QuickCheck >=2.13.2,
+        QuickCheck,
```

Are you sure it is safe to remove this lower bound?

Lower bounds are usually added for a reason.

```suggestion
        QuickCheck >=2.13.2,
```

---

Good catch! I shouldn't have removed that.